deprecated/postgresql-apb/roles/postgresql-apb-openshift/tasks/main.yml

```diff
@@ -88,5 +88,5 @@
   register: encoded_bind_credentials
 
 - copy:
-   content="<BIND_CREDENTIALS>{{ encoded_bind_credentials.stdout }}</BIND_CREDENTIALS>"
+   content="{{ encoded_bind_credentials.stdout }}"
```

Comment:
This is ok now because we're writing it to bind-creds which basically substitutes for `<BIND_CREDENTIALS>`